src/logic/collectibles/store/selectors/index.ts

```diff
@@ -16,11 +16,18 @@ export const nftAssetsListSelector = createSelector(nftAssets, (assets): NFTAsse
   return assets ? Object.values(assets) : []
 })
 
+export const availableNftAssetsAddresses = createSelector(nftTokensSelector, (userNftTokens): string[] => {
+  return userNftTokens.map((nftToken) => nftToken.assetAddress)
```

Comment:
won't it possibly return repeated addresses?

Reply:
Yes you were right good catch! I fixed it